doc/whatsnew/2.12.rst

```diff
@@ -205,6 +205,20 @@ Other Changes
 
   Closes #5216
 
+* ``missing-raises-doc`` will now check the class hierarchy of the raised exceptions
```

Comment:
This should go in `2.13` 😅 

Sorry, I totally missed this previously..

Reply:
I hope the just added commit is fine